rewrites/src/main/scala/fix/scala213/ExplicitNullaryEtaExpansion.scala

```diff
@@ -0,0 +1,17 @@
+package fix.scala213
+
+import scala.meta._
+
+import scalafix.v1._
+
+final class ExplicitNullaryEtaExpansion extends SemanticRule("fix.scala213.ExplicitNullaryEtaExpansion") {
+  override def fix(implicit doc: SemanticDocument) = {
+    doc.tree.collect { case eta: Term.Eta =>
+      eta.expr.symbol.info.map(_.signature).collect {
+        case MethodSignature(_, Nil, _) =>
```

Comment:
So for nullary methods that override a nilary one, there's no rewrite because the Scala compiler inserts the `()` parameter list. I guess that's fine, because `f _` is allowed in this case.

Scala 2.13.x with `-Xsource:3`

```
scala> class C { override def toString = "" }
class C

scala> (new C).toString _
val res2: () => String = $Lambda$989/107279519@60ab895f
```

Dotty:

```
scala> class C { override def toString = "" }
// defined class C

scala> (new C).toString _
1 |(new C).toString _
  |^^^^^^^^^^^^^^^^^^
  |Only function types can be followed by _ but the current expression has type String

scala> class D { override def toString() = "" }
// defined class D

scala> (new D).toString _
val res4: () => String = Lambda$1255/1401633928@19b9f903
```

Reply:
Yeah, but we have no override information in rewrites, so can't do anything about that 😞 (see https://github.com/scala/scala-rewrites/pull/14#issuecomment-608355809)